source/edge-server.txt

```diff
@@ -0,0 +1,126 @@
+:template: product-landing
+:hidefeedback: header
+:noprevnext:
+
+.. _edge-server:
+
+=================
+Atlas Edge Server
+=================
+
+.. toctree::
+   :titlesonly:
+   :hidden:
+   
+   Configure Edge Server </edge-server/configure>
+   Connect to Edge Server </edge-server/connect>
+   Use MongoDB APIs with Edge Server </edge-server/mongodb>
+
+.. introduction::
+
+   You have a warehouse, retail store, air-gapped laboratory, or another 
+   premises where devices should be able to seamlessly sync their data.
+   Those locations may or may not be online at any given time. Lack of a 
+   network connection shouldn't prevent the devices from sharing data
+   with each other.
+
+   Edge Server runs on-premises and handles sync between local devices,
+   and bi-directional sync between the Edge Server and Atlas.
+
+   Edge Server is currently in private preview. If you are interested in 
+   previewing the Atlas Device Sync Edge Server, complete this form.
+
+   .. button:: I'm interested!
```

Comment:
this arrow icon in this button looks kind of weird
<img width="137" alt="image" src="https://github.com/mongodb/docs-app-services/assets/3311964/ea1a8386-7739-4a46-8981-9ac828ff84d6">